### PR TITLE
Clean up storage tests to reduce flakiness

### PR DESCRIPTION
**Pull request by awels**
**What this PR does / why we need it**:
Increased some timeouts to allow longer times to finish tests. Cleaned up Ptr from deprecated functions to newer functions. Made condition checks order independent and return better state information if the test does fail.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/reporter/kubernetes.go [hunk 1]
```diff
@@ -171,6 +171,11 @@ func (r *KubernetesReporter) dumpNamespaces(duration time.Duration, vmiNamespace
 	r.logVMExports(virtCli)
 	r.logDeployments(virtCli)
 	r.logDaemonsets(virtCli)
+	r.logVolumeSnapshots(virtCli)
+	r.logVolumeSnapshotContents(virtCli)
+	r.logVolumeSnapshotClasses(virtCli)
+	r.logVirtualMachineSnapshots(virtCli)
+	r.logVirtualMachineSnapshotContents(virtCli)
```
Comment (mhenriks) [inline tests/reporter/kubernetes.go:178]: Can we add DataVolumes to the list?

Comment (awels) [reply tests/reporter/kubernetes.go:178]: https://github.com/kubevirt/kubevirt/blob/main/tests/reporter/kubernetes.go#L170 should already be there.

**Pull request merged**